src/ssh-util/src/tunnel_manager.rs

```diff
@@ -7,7 +7,7 @@
 // the Business Source License, use of this software will be governed
 // by the Apache License, Version 2.0.
 
-//! SSH tunnel management for the storage layer.
+//! _Global_ SSH tunnel management.
```

Comment:
```suggestion
//! SSH tunnel management.
```

(I'm not sure what "global" means in this context; I think it's best to just drop it.)